.github/workflows/python-tests.yml

```diff
@@ -0,0 +1,29 @@
+name: Pytest
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.9]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v2
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install pytest
+          pip install pytest-cov
+      - name: Test with pytest
+        run: |
+          pytest
+      - name: Test coverage
+        run: |
+          pytest --cov=.
```

Comment:
You might want to combine these two steps into one, so you don't run the tests twice!